### PR TITLE
Add alt text for episodes 3-7

### PR DESCRIPTION
**Pull request by ErinBecker**
@abostroem or @rudyphd please check and make sure the interpretations I've added in the alt-text of the graphs is accurate. It should be of the format 

alt= “`Chart type` of `type of data` where `reason for including chart`” as suggested by Amy Cesal in [this article](https://medium.com/nightingale/writing-alt-text-for-data-visualization-2a218ef43f81).

### CODE REVIEW

**Comment by ErinBecker** [comment]
Ready for review. Please make any necessary edits and go ahead and merge!

**Review by abostroem — COMMENTED** [review]
- I've suggested text for most of these - feel free to reject or modify if its too wordy. 
- It also feels really informal to me to say "The Price-Whelan and Bonaca paper" maybe we can say "The Price-Whelan and Bonaca (2018) paper" or "Price-Whelan and Bonaca (2018)"
- There are also a mix of full and relative urls - I'm not sure if the full urls will cause us trouble down the line with template updates

File: _episodes/03-motion.md [hunk 1]
```diff
@@ -475,7 +475,7 @@ plt.ylabel('phi2 (degree GD1)');
 ~~~
 {: .output}
    
-![png](../fig/03-motion_files/03-motion_43_0.png)
+![Scatter plot of phi1 versus phi2 in GD-1 coordinates, showing selected region is rectangular.](../fig/03-motion_files/03-motion_43_0.png)
```
Comment (rudyphd) [inline _episodes/03-motion.md:478]:
```suggestion
![Scatter plot of phi1 vs phi2, showing selected region is rectangular in the GD-1 reference frame.](../fig/03-motion_files/03-motion_43_0.png)
```

File: _episodes/03-motion.md [hunk 2]
```diff
@@ -766,7 +766,7 @@ plt.ylabel('Proper motion phi2 (mas/yr GD1 frame)');
 ~~~
 {: .output}
  
-![png](../fig/03-motion_files/03-motion_67_0.png)
+![Scatter plot of proper motion in GD-1 frame of selected stars showing most are near the origin.](../fig/03-motion_files/03-motion_67_0.png)
```
Comment (rudyphd) [inline _episodes/03-motion.md:769]:
```suggestion
![Scatter plot of proper motion in GD-1 frame of selected stars showing the need to zoom in on the origin where most of the stars are located.](../fig/03-motion_files/03-motion_67_0.png)
```

File: _episodes/03-motion.md [hunk 3]
```diff
@@ -791,7 +791,7 @@ plt.ylim(-10, 10);
 ~~~
 {: .output}
     
-![png](../fig/03-motion_files/03-motion_69_0.png)
+![Zoomed in view of previous scatter plot showing overdense region.](../fig/03-motion_files/03-motion_69_0.png)
```
Comment (rudyphd) [inline _episodes/03-motion.md:794]:
```suggestion
![Zoomed in view of previous scatter plot showing that the overdose region is not yet visible with our selection.](../fig/03-motion_files/03-motion_69_0.png)
```

File: _episodes/03-motion.md [hunk 4]
```diff
@@ -806,7 +806,7 @@ As we can see in the following figure, many stars in GD-1 are less
 than 1 degree from the line `phi2=0`.
 
 <img
-src="https://github.com/datacarpentry/astronomy-python/raw/gh-pages/fig/gd1-4.png">
+src="https://github.com/datacarpentry/astronomy-python/raw/gh-pages/fig/gd1-4.png" alt="Scatter plot with selection on proper motion and photometry showing many stars in GD-1 are within 1 degree of phi2 = 0.>
```
[the remark this answers is not in the document]
Comment (rudyphd) [reply _episodes/03-motion.md:809]: > Should alt text mention that this is a panel from Fig 1 in the original paper?

I don't think this is necessary in the alt text.

File: _episodes/03-motion.md [hunk 5]
```diff
@@ -731,7 +731,7 @@ Price-Whelan and Bonaca paper, the one that shows components of proper
 motion as a scatter plot:
 
 <img width="300"
-src="https://github.com/datacarpentry/astronomy-python/raw/gh-pages/fig/gd1-1.png">
+src="https://github.com/datacarpentry/astronomy-python/raw/gh-pages/fig/gd1-1.png" alt="Scatter of proper motion phi1 versus phi2 showing overdensity in negative proper motions of GD-1 stars.">
```
[the remark this answers is not in the document]
Comment (rudyphd) [reply _episodes/03-motion.md:734]: > Should we mention that this is a panel for Fig 1 in the original paper?

I don't think this is necessary in the alt text.

File: _episodes/03-motion.md [hunk 6]
```diff
@@ -973,7 +973,7 @@ plot_proper_motion(centerline_df)
 ~~~
 {: .output}
    
-![png](../fig/03-motion_files/03-motion_92_0.png)
+![Scatter plot of proper motion of selected stars showing cluster near (-7.5, 0).](../fig/03-motion_files/03-motion_92_0.png)
```
Comment (rudyphd) [inline _episodes/03-motion.md:976]:
```suggestion
![Scatter plot of proper motion in GD-1 frame of stars selected near the center line of GD-1 showing the overdense region near (-7.5, 0) which represents the proper motion of stars in GD-1.](../fig/03-motion_files/03-motion_92_0.png)
```

File: _episodes/03-motion.md [hunk 7]
```diff
@@ -1041,7 +1041,7 @@ plt.plot(pm1_rect, pm2_rect, '-');
 ~~~
 {: .output}
    
-![png](../fig/03-motion_files/03-motion_100_0.png)
+![Scatter plot of proper motion with blue box showing overdense region selected for our analysis.](../fig/03-motion_files/03-motion_100_0.png)
```
Comment (rudyphd) [inline _episodes/03-motion.md:1044]:
```suggestion
![Scatter plot of proper motion in the GD-1 frame showing that our rectangle surrounds the overdense region we wish to select for our analysis.](../fig/03-motion_files/03-motion_100_0.png)
```

File: _episodes/03-motion.md [hunk 8]
```diff
@@ -1111,7 +1111,7 @@ plt.ylabel('phi2 (degree GD1)');
 ~~~
 {: .output}
    
-![png](../fig/03-motion_files/03-motion_110_0.png)
+![Scatter plot of coordinates of stars in selected region, showing tidal stream.](../fig/03-motion_files/03-motion_110_0.png)
```
Comment (rudyphd) [inline _episodes/03-motion.md:1114]:
```suggestion
![Scatter plot of coordinates of stars in the GD-1 frame that have been selected to have proper motion in the overdense region, showing an overdensity of stars in the tidal stream GD-1.](../fig/03-motion_files/03-motion_110_0.png)
```

File: _episodes/04-select.md [hunk 9]
```diff
@@ -174,7 +174,7 @@ plt.plot(x, y, 'gx', markersize=0.3, alpha=0.3);
 ~~~
 {: .output}
  
-![png](../fig/04-select_files/04-select_14_0.png)
+![Proper motion of stars in GD-1, showing selected region as blue box and stars within selection as green points.](../fig/04-select_files/04-select_14_0.png)
```
Comment (rudyphd) [inline _episodes/04-select.md:177]:
```suggestion
![Scatter plot of proper motion in the GD-1 frame with the proper motion selection rectangle plotted as a blue box and the proper motion of the stars selected to be inside the rectangle plotted as green points showing that we are selecting the correct points.](../fig/04-select_files/04-select_14_0.png)
```

File: _episodes/04-select.md [hunk 10]
```diff
@@ -201,7 +201,7 @@ plt.ylim([-20, 5]);
 ~~~
 {: .output}
   
-![png](../fig/04-select_files/04-select_16_0.png)
+![Proper motion in ICRS frame, showing selected stars are more spread out in this frame.](../fig/04-select_files/04-select_16_0.png)
```
Comment (rudyphd) [inline _episodes/04-select.md:204]:
```suggestion
![Scatter plot of proper motion in the ICRS frame with the proper motion selected stars plotted as green points showing that the region is no longer rectangular in the ICRS frame.](../fig/04-select_files/04-select_16_0.png)
```

File: _episodes/04-select.md [hunk 11]
```diff
@@ -340,7 +340,7 @@ plt.ylim([-20, 5]);
 ~~~
 {: .output}
     
-![png](../fig/04-select_files/04-select_29_0.png)
+![Proper motion in ICRS, with convex hull shown as blue boundary and selected stars as green points.](../fig/04-select_files/04-select_29_0.png)
```
Comment (rudyphd) [inline _episodes/04-select.md:343]:
```suggestion
![Scatter plot of proper motion in the ICRS frame with the proper motion selected stars plotted as green points and the convex hull as a blue boundary, showing that the convex hull outlines the selected stars.](../fig/04-select_files/04-select_29_0.png)
```

File: _episodes/04-select.md [hunk 12]
```diff
@@ -646,7 +646,7 @@ plt.ylabel('dec (degree ICRS)');
 ~~~
 {: .output}
     
-![png](../fig/04-select_files/04-select_66_0.png)
+![Scatter plot of right ascension and declination of selected stars in ICRS frame.](../fig/04-select_files/04-select_66_0.png)
```
Comment (rudyphd) [inline _episodes/04-select.md:649]:
```suggestion
![Scatter plot of right ascension and declination of selected stars in ICRS frame showing that the selected region is not rectangular as that the overdensity of stars in GD-1 is visible with our selection.](../fig/04-select_files/04-select_66_0.png)
```

File: _episodes/04-select.md [hunk 13]
```diff
@@ -678,14 +678,14 @@ plt.ylabel('phi2 (degree GD1)');
 ~~~
 {: .output}
   
-![png](../fig/04-select_files/04-select_72_0.png)
+![Scatter plot of phi1 versus phi2 in GD-1 frame after selecting on proper motion.](../fig/04-select_files/04-select_72_0.png)
```
Comment (rudyphd) [inline _episodes/04-select.md:681]:
```suggestion
![Scatter plot of phi1 and phi2 of selected stars in the GD-1 frame showing that the selected region is rectangular in this reference frame and that the overdensity of stars in GD-1 is visible with our selection.](../fig/04-select_files/04-select_72_0.png)
```

File: _episodes/05-join.md [hunk 14]
```diff
@@ -31,7 +31,7 @@ The following figure from the paper is a color-magnitude diagram for
 the stars selected based on proper motion:
 
 <img width="300"
-src="https://github.com/datacarpentry/astronomy-python/raw/gh-pages/fig/gd1-3.png">
+src="https://github.com/datacarpentry/astronomy-python/raw/gh-pages/fig/gd1-3.png" alt="Color-magnitude diagram for the stars selected based on proper motion, from Price-Whelan and Bonaca paper.">
```
Comment (rudyphd) [inline _episodes/05-join.md:34]:
```suggestion
src="https://github.com/datacarpentry/astronomy-python/raw/gh-pages/fig/gd1-3.png" alt="Color-magnitude diagram for the stars selected based on proper motion, from Price-Whelan & Bonaca, 2018 paper showing that the putative the GD-1 stars cluster around single isochrone, shown in red.">
```

File: _episodes/05-join.md [hunk 15]
```diff
@@ -361,7 +361,7 @@ If you are curious, you can [read more about
 it](https://chartio.com/learn/databases/how-does-indexing-work/).
 
 <img
-src="https://github.com/datacarpentry/astronomy-python/raw/gh-pages/fig/join.png">
+src="https://github.com/datacarpentry/astronomy-python/raw/gh-pages/fig/join.png" alt="Diagram showing relationship between the gaia_source, panstarrs1_best_neighbor, and panstarrs1_original_valid tables and result table.">
```
Comment (rudyphd) [inline _episodes/05-join.md:364]:
```suggestion
src="https://github.com/datacarpentry/astronomy-python/raw/gh-pages/fig/join.png" alt="Diagram showing relationship between the gaia_source, panstarrs1_best_neighbor, and panstarrs1_original_valid tables and the table that results from joining gaia_source to panstarrs1_best_neighbor and joining again to panstarrs1_original_valid.">
```

File: _episodes/06-photo.md [hunk 16]
```diff
@@ -34,7 +34,7 @@ The following figure from the paper is a color-magnitude diagram
 showing the stars we previously selected based on proper motion:
 
 <img width="300"
-src="https://github.com/datacarpentry/astronomy-python/raw/gh-pages/fig/gd1-3.png">
+src="https://github.com/datacarpentry/astronomy-python/raw/gh-pages/fig/gd1-3.png" alt="Color-magnitude diagram for the stars selected based on proper motion, from Price-Whelan and Bonaca paper.">
```
Comment (rudyphd) [inline _episodes/06-photo.md:37]:
```suggestion
src="https://github.com/datacarpentry/astronomy-python/raw/gh-pages/fig/gd1-3.png" alt="Color-magnitude diagram for the stars selected based on proper motion, from Price-Whelan & Bonaca, 2018 paper showing that the putative the GD-1 stars cluster around single isochrone, shown in red.">
```

File: _episodes/06-photo.md [hunk 17]
```diff
@@ -91,7 +91,7 @@ diagram](https://en.wikipedia.org/wiki/Galaxy_color%E2%80%93magnitude_diagram)
 from the original paper:
 
 <img width="300"
-src="https://github.com/datacarpentry/astronomy-python/raw/gh-pages/fig/gd1-3.png">
+src="https://github.com/datacarpentry/astronomy-python/raw/gh-pages/fig/gd1-3.png" alt="Color-magnitude diagram for the stars selected based on proper motion, from Price-Whelan and Bonaca paper.">
```
Comment (rudyphd) [inline _episodes/06-photo.md:94]:
```suggestion
src="https://github.com/datacarpentry/astronomy-python/raw/gh-pages/fig/gd1-3.png" alt="Color-magnitude diagram for the stars selected based on proper motion, from Price-Whelan & Bonaca, 2018 paper showing that the putative the GD-1 stars cluster around single isochrone, shown in red.">
```

File: _episodes/06-photo.md [hunk 18]
```diff
@@ -177,7 +177,7 @@ plot_cmd(candidate_df)
 ~~~
 {: .output}
     
-![png](../fig/06-photo_files/06-photo_12_0.png)
+![Color magnitude diagram of our selected stars showing overdense region in lower left.](../fig/06-photo_files/06-photo_12_0.png)
```
Comment (rudyphd) [inline _episodes/06-photo.md:180]:
```suggestion
![Color magnitude diagram of our selected stars showing the main sequence of GD-1 as an overdensity in the lower left.](../fig/06-photo_files/06-photo_12_0.png)
```

File: _episodes/06-photo.md [hunk 19]
```diff
@@ -540,7 +540,7 @@ plt.legend();
 ~~~
 {: .output}
     
-![png](../fig/06-photo_files/06-photo_62_0.png)
+![Color magnitude diagram of our selected stars showing left boundary as blue curve and right boundary as orange curve.](../fig/06-photo_files/06-photo_62_0.png)
```
Comment (rudyphd) [inline _episodes/06-photo.md:543]:
```suggestion
![Color magnitude diagram of our proper motion selected stars showing left boundary of the region around the GD-1 main sequence as a blue curve and the right boundary of the region as an orange curve.](../fig/06-photo_files/06-photo_62_0.png)
```

File: _episodes/06-photo.md [hunk 20]
```diff
@@ -620,7 +620,7 @@ plt.plot(color_loop, mag_loop);
 ~~~
 {: .output}
     
-![png](../fig/06-photo_files/06-photo_70_0.png)
+![Color magnitude diagram of our selected stars showing polygon defined by boundaries as blue curve.](../fig/06-photo_files/06-photo_70_0.png)
```
Comment (rudyphd) [inline _episodes/06-photo.md:623]:
```suggestion
![Color magnitude diagram of our proper motion selected stars showing the polygon we defined around the main sequence of GD-1 as a blue curve.](../fig/06-photo_files/06-photo_70_0.png)
```

File: _episodes/06-photo.md [hunk 21]
```diff
@@ -797,7 +797,7 @@ plt.plot(x, y, 'go', markersize=0.5, alpha=0.5);
 ~~~
 {: .output}
     
-![png](../fig/06-photo_files/06-photo_91_0.png)
+![Color magnitude diagram of our selected stars showing polygon as orange curve with isochrone overlaid as blue curve.](../fig/06-photo_files/06-photo_91_0.png)
```
Comment (rudyphd) [inline _episodes/06-photo.md:800]:
```suggestion
![Color magnitude of our proper motion selected stars showing the polygon around the main sequence of GD-1 as an orange curve with the isochrone overlaid as a blue curve, and demonstrating thatthe photometry and proper motion selected stars which are shown as green points are inside the polygon.](../fig/06-photo_files/06-photo_91_0.png)
```

File: _episodes/06-photo.md [hunk 22]
```diff
@@ -823,7 +823,7 @@ plt.axis('equal');
 ~~~
 {: .output}
   
-![png](../fig/06-photo_files/06-photo_93_0.png)
+![Right ascension and declination of selected stars in GD-1 frame after selecting for both proper motion and photometry.](../fig/06-photo_files/06-photo_93_0.png)
```
Comment (rudyphd) [inline _episodes/06-photo.md:826]:
```suggestion
![Phi1 and phi2 of selected stars in GD-1 frame after selecting for both proper motion and photometry showing that with this selection the GD-1 stream is now clearly visible.](../fig/06-photo_files/06-photo_93_0.png)
```

File: _episodes/07-plot.md [hunk 23]
```diff
@@ -177,7 +177,7 @@ plot_second_selection(winner_df)
 ~~~
 {: .output}
     
-![png](../fig/07-plot_files/07-plot_13_0.png)
+![On sky coordinates of our selected stars after filtering on proper motion and photometry.](../fig/07-plot_files/07-plot_13_0.png)
```
Comment (rudyphd) [inline _episodes/07-plot.md:180]:
```suggestion
![Phi1 and phi2 of selected stars in GD-1 frame after selecting for both proper motion and photometry showing that with this selection the GD-1 stream is now clearly visible.](../fig/07-plot_files/07-plot_13_0.png)
```

File: _episodes/07-plot.md [hunk 24]
```diff
@@ -568,7 +568,7 @@ plot_proper_motion(centerline_df)
 ~~~
 {: .output}
    
-![png](../fig/07-plot_files/07-plot_53_0.png)
+![Proper motion with overlaid polygon showing our selected stars.](../fig/07-plot_files/07-plot_53_0.png)
```
Comment (rudyphd) [inline _episodes/07-plot.md:571]:
```suggestion
![Proper motion in phi1 and phi2 overlaid with orange filled polygon showing stars selected to be in GD-1 based on overdensity of non-zero phi1 proper motion.](../fig/07-plot_files/07-plot_53_0.png)
```

File: _episodes/07-plot.md [hunk 25]
```diff
@@ -651,7 +651,7 @@ plot_cmd(candidate_df)
 ~~~
 {: .output}
     
-![png](../fig/07-plot_files/07-plot_63_0.png)
+![Color magnitude diagram of the stars we've selected by proper motion.](../fig/07-plot_files/07-plot_63_0.png)
```
Comment (rudyphd) [inline _episodes/07-plot.md:654]:
```suggestion
![Color-magnitude diagram for the stars selected based on proper motion, from Price-Whelan and Bonaca (2018) paper showing that the main sequence of GD-1 is visible as an overdensity in the lower left of the figure.](../fig/07-plot_files/07-plot_63_0.png)
```

File: _episodes/07-plot.md [hunk 26]
```diff
@@ -788,7 +734,7 @@ plt.tight_layout()
 ~~~
 {: .output}
    
-![png](../fig/07-plot_files/07-plot_69_0.png)
+![Four paneled plot showing our first recreation of figure 1 from the Price-Whelan and Bonaca paper.](../fig/07-plot_files/07-plot_69_0.png)
```
Comment (rudyphd) [inline _episodes/07-plot.md:737]:
```suggestion
![Four paneled plot showing our first recreation of figure 1 from the Price-Whelan and Bonaca (2018) paper with equal sized panels.](../fig/07-plot_files/07-plot_69_0.png)
```

**Comment by rudyphd** [comment]
Alt text resolved. 

> * There are also a mix of full and relative urls - I'm not sure if the full urls will cause us trouble down the line with template updates

I've added an issue to address this question. I'm going to merge this PR.
